checks/system/win32.py

```diff
@@ -54,12 +54,12 @@ def check(self, agentConfig):
                 u"Timeout while querying Win32_PerfRawData_PerfOS_System WMI class."
                 u" Processes metrics will be returned at next iteration."
             )
-            return
+            return []
 
         if not (len(self.wmi_sampler)):
             self.logger.info('Missing Win32_PerfRawData_PerfOS_System WMI class.'
```

Comment:
Can we take advantage of this PR to change these log levels to `warning` instead of `info` ? What do you think ?